src/main/java/lotto/domain/LottoShop.java

```diff
@@ -0,0 +1,36 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.IntStream;
+
+public class LottoShop {
+	public static final int LOTTO_PRICE = 1000;
```

Comment:
https://github.com/next-step/java-lotto-pro/pull/241#discussion_r745674279
금액을 의미하는 상수 또한 Money 객체로 래핑할 수 있을 것 같아요 :)

```suggestion
	public static final Money LOTTO_PRICE = new Money(1000);
```